data/exclusions-disease-gene.tsv

```diff
@@ -0,0 +1,8 @@
+omim_id	mondo_id	mondo_label	orcid	exclusion_reason_comment
+OMIM:603956	MONDO:0002974	cervical cancer'	https://orcid.org/0000-0002-4142-7153	evidence of various genes involved
+OMIM:619151	MONDO:0030894	"AMED syndrome, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:158901	MONDO:0008031		https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:108770	MONDO:0007171	atrial standstill 1'	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:620040	MONDO:0031057	"dyskeratosis congenita, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:619478	MONDO:0030355	"facioscapulohumeral muscular dystrophy 4, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:300818	MONDO:0010438	paroxysmal nocturnal hemoglobinuria 1	https://orcid.org/0000-0002-4142-7153	"disease caused by a somatic mutation, therefore a gene association stating this is due to a germline mutation should not be added"
```

Comment:
There are no disease to gene associations any of these diseases (OMIM phenotypes) now.

Reply:
Indeed!
Just for clarification, you mean no more _disease-defining_ (`RO:0004013`) associations. They are now `RO:0003302`.